spec/support/fake_analytics.rb

```diff
@@ -1,4 +1,31 @@
 class FakeAnalytics
+  PiiDetected = Class.new(StandardError)
```

Comment:
Nitpick / question: Would it make sense for this to be part of the `PiiAlerter` module namespace?

Reply:
To me, `FakeAnalytics::PiiAlerter::PiiDetected` seemed redundant so that's why I shortened it like this 🤷

Reply:
Fair enough!